File: UI/EAgroUIApp/src/app/merchant/shipment-item-details.ts
export class ShipmentItemDetails {
    constructor(
        public id: number,
        public collectionCenterId: number,
        public collectionCenterName: string,
        public farmerId: number,
        public cropName: string,
        public grade: string,
        public containerType: string,
        public quantity: number,
        public totalWeight: number,
        public netWeight: number,
        public collectionDate: string

    ) { }
}

File: API/Shipments/Models/ShipmentItemDetails.cs
namespace Shipments.Models
{
    public class ShipmentItemDetails
    {
        public int Id { get; set; }
        public int CollectionCenterId { get; set; }
        public int FarmerId { get; set; }
        public string CropName { get; set; }
        public string Grade { get; set; }
        public string ContainerType { get; set; }
        public int Quantity { get; set; }
        public double TotalWeight { get; set; }
        public double NetWeight { get; set; }
        public DateTime CollectionDate { get; set; }
    }
}